src/processTargets.ts

```diff
@@ -474,9 +563,12 @@ function getLineSelectionContext(
     selection.editor.document.lineAt(end.line).range.end.character
   );
 
+  const isInDelimitedList =
+    leadingDelimiterRange != null || trailingDelimiterRange != null;
+
   return {
-    isInDelimitedList: true,
-    containingListDelimiter: "\n",
+    isInDelimitedList,
+    containingListDelimiter: isInDelimitedList ? "\n" : undefined,
```

Comment:
should be 
```suggestion
    containingListDelimiter: isInDelimitedList ? "\n\n" : undefined,
```

no?

Reply:
Agreed is just left over from the line context that I copied